### DISCUSSION TITLE
Proximity awareness?

### DISCUSSION THREAD

**Discussion by clee** [mikebrady/shairport-sync#1842] [Ideas]
I have a Sonos soundbar that supports Airplay, and it has a really neat feature: when I'm physically close to it and I open the Music app on my iPhone, it automatically detects that the soundbar is nearby and shows a prompt asking me if I want to play my music on it. I'm not sure how this works, though. Maybe something related to "Airplay discoverability over bluetooth"? https://www.macrumors.com/2014/03/12/apple-tv-6-1-bluetooth-airplay/

It would be really cool if I could configure shairport-sync to support the same proximity discoverability feature so that I'm prompted to send my music to the shairport-sync server when I'm nearby.

**Comment by mikebrady** [comment]
Thanks. It's a nice idea for sure, but unfortunately Apple have not released details of the protocols that are used to implement this, so we are stymied.

**Reply by clee** [reply]
Ah, that makes sense. Oh well, thanks for responding at least!
